data/changelog.txt

```diff
@@ -445,8 +252,9 @@ compose.py: allow default sized tilesheets to specify offsets.
 Enable compiling an Android build under Windows.
 Tools: add a Python script to reformat vehicles to use parts arrays.
 Supress Homebrew update for faster OSX builds
-Add a problem matcher to match json errors during unit test
-Update HeaderGuardCheck.cpp and fix-compilation-database.py to support custom build directory
+Add a problem matchers for json errors, compiler errors, DebugLog emission and Catch2 errors.
```

Comment:
Suggestion: "Add problem matchers for json errors, compiler errors, DebugLog emission and Catch2 errors."